contrib/debian/fwupd-qubes-vm-whonix.postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+FWUPD_UPDATEVM_DIR=/home/user/.cache/fwupd
+FWUPD_UPDATEVM_SCRIPT=/usr/share/qubes-fwupd/fwupd_download_updates.py
+
+mkdir $FWUPD_UPDATEVM_DIR
+chown -R user:user $FWUPD_UPDATEVM_DIR
+chmod +x $FWUPD_UPDATEVM_SCRIPT
```

Comment:
If it needs to be executable and coming out of `/usr/share` I think it's probably installed to the wrong location.  Shouldn't it be in `/usr/bin/` or `/usr/sbin/` then?

Reply:
I've moved main executable script (`qubes-fwupdmgr`) to the `/usr/sbin`, also I set read-only permissions for `/usr/share` scripts.